### ISSUE TITLE
172104914 Funny dates (Michael)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1341]
Issue reported by **LINCC** at 2020-06-14 22:55:14 UTC
Time: 2020-06-14T22:44:58+00:00
Session ID: 43eb409306732ce54a2d925d295dff34
Problem Page URL: [/freereg1_csv_files/5463bf64e937902782544a00?locale=en](/freereg1_csv_files/5463bf64e937902782544a00?locale=en)
Previous Page URL: [https://www.freereg.org.uk/registers/540a04ddeca9ebd28ed283a6?locale=en](https://www.freereg.org.uk/registers/540a04ddeca9ebd28ed283a6?locale=en)
Reported Issue:
See the attached.

24 records in file. Start date is 2020. Last date is 1300. Every decadel is zero. File consists of  entries with only birth dates (no baptism dates).

I noticed this problem and carried out a Reprocess. This is the result of the Reprocess. It is the same as before the Reprocess.
https://www.freereg.org.uk/uploads/feedback/screenshot/5ee6aad2f493fd017e9ad0dd/Capture.JPG    

![Screenshot](https://www.freereg.org.uk)

E0 U2 S6 C6: P14

**Comment by SteveBiggs** [comment]
Shouldn't a reprocess sort this out?

**Comment by Sherlock21** [comment]
Michael: remember that I am deleting a fair number if files for you. -
All done now.

**Comment by Captainkirkdawson** [comment]
Suspect it may be an artifact of there being only a single decade

**Comment by Captainkirkdawson** [comment]
Yes @SteveBiggs it should yes but does not. Needs further review

**Comment by richpomfret** [comment]
@PatReynolds to add in priority - doesn't impact upon many.

**Comment by richpomfret** [comment]
@edickens to take a look through the file and see if there's anything there.

**Comment by edickens** [comment]
I'm mystified.  I downloaded the file then uploaded with my UserID and it has worked fine.  See 1804 REGManager.

![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/c40ead60-1155-4cd7-82c8-16a08ae3f6f2)

But reprocessing does not work on the original.

The  UsereID LCD is active, so it is not to do with that.

Could it be a hidden character?

**Comment by edickens** [comment]
Do you need me to do any more with this?
It looks as if the Reprocess has a different way of calculating the year range to Upload.
This file is not all one year but is all Birth dates.
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/914b8b15-d2d2-4ef5-8a28-5bc9075f6747)

**Comment by PatReynolds** [comment]
Pat to contact Mike to see if it is part of the Baptisms without date of Baptism problem - there are two Baptist churches in Spaliding with about 100 entries total.

**Comment by PatReynolds** [comment]
Pat to contact Mike to see if it is part of the Baptisms without date of Baptism problem - there are two Baptist churches in Spaliding with about 100 entries total.

**Comment by edickens** [comment]
Not sure why this is assigned to me.

**Comment by SteveBiggs** [comment]
Given Eric's earlier test it appears to need a programmer to analyse what the cause might be.

**Comment by DeniseColbert** [comment]
To retest when 1300 year issue is addressed

**Comment by stsccfr** [comment]
I could not reproduce this on test3.  I uploaded the original file and reprocessed it, but the Show File details always showed the correct min and max values, namely 1804 and 1809.  I can't find any code changes in the last 5 years that would have fixed this, so I don't know when or how it got fixed.

**Comment by stsccfr** [comment]
Closing as no longer reproducible
